xenon/cogs/backups.py

```diff
@@ -108,7 +108,7 @@ def random_id(self):
 
         __Arguments__
 
-        **backup_id**: The id of the backup or the guild id for the latest automated backup
+        **backup_id**: The id of the backup or the guild id of the latest automated backup
```

Comment:
Still I don't really like the sentence… :thinking:

Reply:
Can you think of a better wording?

Reply:
@niansa I can't think of anything better, any ideas?